cmd/tsbs_load_mongo/main.go

```diff
@@ -73,15 +79,14 @@ func init() {
 	retryableWrites = viper.GetBool("retryable-writes")
 	orderedInserts = viper.GetBool("ordered-inserts")
 	randomFieldOrder = viper.GetBool("random-field-order")
-	timeseriesCollectionSharded = viper.GetBool("timeseries-collection-sharded")
+	collectionSharded = viper.GetBool("collection-sharded")
+	numInitChunks = viper.GetUint("number-initial-chunks")
+	shardKeySpec = viper.GetString("shard-key-spec")
+	loadBalancerOn = viper.GetBool("load-balancer-on")
```

Comment:
nit: Suggest replacing `load balancer` with `balancer` to be consistent with the terminology that we use everywhere else in the code and docs.

Reply:
done